pkg/controller/controller.go

```diff
@@ -187,6 +191,23 @@ func (c *Controller) ensureK8sResources(funcObj *kubelessApi.Function) error {
 	}
 	funcObj.ObjectMeta.Labels["function"] = funcObj.ObjectMeta.Name
 
+	controllerNamespace := os.Getenv("KUBELESS_INSTALLED_NAMESPACE")
+	cm, _ := c.clientset.CoreV1().ConfigMaps(controllerNamespace).Get("kubeless-config", metav1.GetOptions{})
```

Comment:
if we have the namespace as env var we can have the config name as one as well. BTW, this conflicts with @rakesh-garimella work in #575 ([link](https://github.com/kubeless/kubeless/pull/575/files#diff-3c862eb54a8e0e161b534e0c67e5379eR93)) so we should make some merging work at some point.

Reply:
added `KUBELESS_CONFIG` to env